_episodes/03-dependency_graphs.md

```diff
@@ -21,21 +21,29 @@ keypoints:
 ---
 
 ## Multi-Step Workflow
-In the previous episode a single step workflow was shown. To make a multi-step workflow, you add more entries to the `steps` field.
-In this episode, the workflow is extended with the next two steps of the RNA-sequencing analysis.
-The next two steps are triming the reads and alignment of the trimmed reads.
+In the previous episode a single step workflow was shown, carrying out a simple RNA read
+of the fruitfly genome. In this episode, the workflow is extended with an equivalent reverse
+RNA read, and the next two steps of the RNA-sequencing analysis, trimming the reads and
+aligning the trimmed reads, are added.
```

Comment:
We've got a bit of an excess of commas in this sentence - I think this would read better:
> the next two steps of the RNA-sequencing analysis **are added**, trimming ....